### PR TITLE
Rename "LDAP" to "LDAP/AD-integration"

### PR DESCRIPTION
**Pull request by MariusBluem**
@nextcloud/ldap

Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: apps/user_ldap/templates/settings.php [hunk 1]
```diff
@@ -55,7 +55,7 @@
 ?>
 
 <form id="ldap" class="section" action="#" method="post">
-	<h2><?php p($l->t('LDAP')); ?></h2>
+	<h2><?php p($l->t('LDAP/AD-integration')); ?></h2>
```
Comment (MorrisJobke) [inline apps/user_ldap/templates/settings.php]: remove the dash please

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

File: apps/user_ldap/templates/settings.php (hunk 1 above)
Comment (MariusBluem) [reply apps/user_ldap/templates/settings.php]: We do it in the same way in the left menu :)

Comment (MorrisJobke) [reply apps/user_ldap/templates/settings.php]: Then fix it there too I would say 😆

Comment (MariusBluem) [reply apps/user_ldap/templates/settings.php]: 👍

File: apps/user_ldap/templates/settings.php [hunk 2]
```diff
@@ -55,7 +55,7 @@
 ?>
 
 <form id="ldap" class="section" action="#" method="post">
-	<h2><?php p($l->t('LDAP')); ?></h2>
+	<h2><?php p($l->t('LDAP / AD integration')); ?></h2>
```
Comment (MorrisJobke) [inline apps/user_ldap/templates/settings.php]: Fixed to be the same everywhere: see https://github.com/nextcloud/server/blob/0eebff152a177dd59ed8773df26f1679f8a88e90/apps/user_ldap/lib/Settings/Section.php#L64

**Review by MorrisJobke — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]

**Pull request merged**